.github/workflows/release.yml

```
@@ -0,0 +1,96 @@
+name: release
+
+on:
+  pull_request:
+    types: [closed]
+
+jobs:
+  patch-release:
+    name: Create Patch Release
+    if: (github.event.pull_request.merged == true && contains(github.event.pull_request.labels.*.name, 'patch') )
+    runs-on: ubuntu-latest
+    steps:
+      # Check out the `master` branch
+      - uses: zendesk/checkout@v2
```

```suggestion
      - uses: zendesk/checkout@v3
```
(and below)

This is either the Node.js deprecation or the save-state/set-output one lol